tests/e2e/playwright.config.ts

```
@@ -40,37 +26,27 @@ const config: PlaywrightTestConfig = {
-        /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
-        trace: 'off',
+        actionTimeout: 0,
 
-        // Browser options
         headless: true,
 
-        // Context options
         viewport: { width: 1280, height: 720 },
         
         // Ignore SSL certificate errors for local development
         ignoreHTTPSErrors: true,
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)